### PR TITLE
initial version

### PR DESCRIPTION
**Pull request by aeh**

### CODE REVIEW

File: test/00_setup.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+const server = http.createServer(function onRequest (req, res) {
+  serve(req, res, finalhandler(req, res));
+});
+
+before(async function () {
+  const { address, port } = await new Promise((resolve, reject) => {
+    server.listen(0, 'localhost', () => resolve(server.address()));
+  });
+  this.host = `http://${address}:${port}`;
+});
```
Comment (timlesallen) [inline test/00_setup.js:19]: I guess this is fine, I'd generally be inclined just to `nock` these out.  Then each test would simply have sth like:
```js
nock('https://thing/blerg.opus').replyWithFile(200, path.join(__dirname, 'blerg.opus'));
```
And you'd have the `clearAll()` or whatever it is for `nock` in an `afterEach()`.

Comment (timlesallen) [reply test/00_setup.js:19]: But either way... this is more realistic I guess.

File: README.md [hunk 2]
```diff
@@ -13,17 +13,20 @@ $ npm install get-media-dimensions
...
+// get image dimensions = { width, height }
+const dimensions = await getMediaDimensions('./image.jpg', 'image');
+const dimensions = await getMediaDimensions('https://somewhere.com/image.jpg', 'image');
 
-Note: To regenerate this section from the jsdoc run `npm run docs` and paste
-the output above.
+// get audio dimensions = { duration }
+const dimensions = await getMediaDimensions('./audio.mp3', 'audio');
+const dimensions = await getMediaDimensions('https://somewhere.com/audio.mp3', 'audio');
+```
```
Comment (timlesallen) [inline README.md]: I do quite like having the kind of output you get from `jsdoc2markdown` as it has parameter types and, importantly here, output type?

**Comment by timlesallen** [comment]
nice work!  One request for doco, one observation that probably isn't significant :)

File: test/00_setup.js (hunk 1 above)
Comment (aeh) [reply test/00_setup.js:19]: @timlesallen the main reason for doing this is that nock does not work for ffprobe calls... I need an actual hosted url to test that properly.

**Pull request merged**